### PR TITLE
feat: Create Confirmation Page | request Price Estimate

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Fix**

This PR solves [CX-2782]

### Description

Added Confirmation Page, Confirmation Page Route and refactored confirmation page component to be able to reuse it
| Web | mWeb |
| --- | --- |
| image | image |

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-2782]: https://artsyproduct.atlassian.net/browse/CX-2782?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/ConfirmationScreenComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Text, Flex, Button } from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+
+export const ConfirmationScreenComponent: React.FC<{
+  title: string
+  subtitle: string
+  buttonTitle: string
```
Comment (MounirDhahri) [inline src/Components/ConfirmationScreenComponent.tsx:7]:
```suggestion
  buttonText: string
```
what do you think about renaming this to `buttonText` 👀?

**Review by MounirDhahri — CHANGES_REQUESTED** [review]
Thanks for getting this done Daria, I left some comments, please let me know if something isn't clear and I can explain it further. 

Also, would you mind if you include screenshots on mobile devices as I think the font sizes are probably different between desktop and mobile

File: src/Apps/MyCollection/myCollectionRoutes.tsx [hunk 2]
```diff
@@ -84,6 +94,15 @@ export const myCollectionRoutes: AppRouteConfig[] = [
       }
     `,
   },
+  {
+    path: "/my-collection/artwork/:artworkID/price-estimate/confirmation",
```
[the remark this answers is not in the document]
Comment (dariakoko) [reply src/Apps/MyCollection/myCollectionRoutes.tsx:98]: Looks good, will use /success

**Review by olerichter00 — APPROVED** [review]
Great work 🌟

**Review by MounirDhahri — APPROVED** [review]
Nice

File: src/Apps/MyCollection/Routes/PriceEstimate/PriceEstimateContactInformation.tsx [hunk 3]
```diff
@@ -86,7 +86,10 @@ export const PriceEstimateContactInformation: React.FC<PriceEstimateContactInfor
         },
       })
 
-      router.push(`/my-collection/artwork/${artwork.internalID}`)
+      router.replace(`/my-collection/artwork/${artwork.internalID}`)
```
[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/Apps/MyCollection/Routes/PriceEstimate/PriceEstimateContactInformation.tsx:89]: +1
was also going to mention this

**Pull request merged**